src/components/material/material-buttons/helper.ts

```diff
@@ -2,10 +2,16 @@ import { intersection } from "lodash";
 import { AccessTypeCode } from "../../../core/dbc-gateway/generated/graphql";
 import { Manifestation } from "../../../core/utils/types/entities";
 
+export const isDigitalArticleService = (manifest: Manifestation) =>
+  manifest.access.some(
+    ({ __typename }) => __typename === "DigitalArticleService"
+  );
+
 export const hasCorrectAccessType = (
   desiredAccessType: AccessTypeCode,
   manifest: Manifestation
 ) => {
+  if (isDigitalArticleService(manifest)) return true;
```

Comment:
Please make this into a separate function that checks for correct access - access and accessType are two different things. 😉